apps/next/src/content/docs/llamaindex/modules/data/memory/index.mdx

```diff
@@ -106,34 +106,41 @@ const memory = createMemory({
 
 Long-term memory is represented as `Memory Block` objects. These objects contain information that are from previous user sessions or from the beginning of the current conversation. When memory is retrieved (by calling `getLLM`), the short-term and long-term memories are merged together within the given `tokenLimit`. 
 
-Currently, there are two predefined memory blocks:
+Currently, there are three predefined memory blocks:
 
 - `staticBlock`: A memory block that stores a static piece of information.
 - `factExtractionBlock`: A memory block that extracts facts from the chat history.
+- `vectorBlock`: A memory block that stores and retrieves chat messages from a vector database using semantic similarity search. Messages are stored individually and retrieved based on their relevance to recent conversation context. Here we've passed in the `vectorStore` and `embedModel` to use to store and retrieve the chat messages.
 
 This sounds a bit complicated, but it's actually quite simple. Let's look at an example:
 
 ```ts
-import { createMemory, factExtractionBlock, staticBlock } from "llamaindex";
+import { createMemory, factExtractionBlock, staticBlock, vectorBlock } from "llamaindex";
+import { QdrantVectorStore } from "@llamaindex/qdrant";
+import { OpenAIEmbedding } from "@llamaindex/openai";
 
 const memoryBlocks= [
   staticBlock({
-    id: "core_info",
     content: "My name is Logan, and I live in Saskatoon. I work at LlamaIndex.",
   }),
   factExtractionBlock({
-    id: "user-extracted_info",
     priority: 1,
     llm: llm,
     maxFacts: 50,
   }),
+  vectorBlock({
+    vectorStore: new QdrantVectorStore({ url: "http://localhost:6333" }),
+    embedModel: new OpenAIEmbedding({ model: "text-embedding-ada-002" }),
```

Comment:
in LITS, a vector store has an `embedModel` - better use it and don't set it in `vectorBlock`